draft-ietf-cdni-request-routing-extensions.xml

```diff
@@ -382,7 +405,110 @@
       </section> 
     </section>
 
-      
+
+    <section anchor="cdni-metadata-ipv4v6cidr-footprint-type" title="CDNI Metadata IPv4v6CIDR Footprint Type">
+      <t> As described in section 5 of <xref target="RFC8008"/>, the FCI Capability Advertisement Object
+          includes an array of CDNI Footprint Objects. Appendix B of <xref target="RFC8008"/> specifies that the
+          semantics of Footprint Objects array is of a multiple, additive, footprint constraints. 
+          Meaning, the advertisement of different footprint types narrows
+          the dCDN's candidacy cumulatively. 
+      </t>
+
+      <t>Sections 4.3.5 and 4.3.6 of <xref target="RFC8006"/> specify the "IPv4CIDR" and "IPv6CIDR"
+          footprint types (respectively), for listing IP addresses blocks. Using Footprint Objects of these
+          types, one can define a FCI Capability Advertisement Object footprint constraints that match IPv4 
+          or IPv6 clients. However, the described "narrowing" semantic of the Footprint Objects array prevents
+          the usage of these objects together in order to create a footprint constraint that matches IPv4 
+          clients together with IPv6 clients. 
+       </t>
+       <t>Below is an example for an attempt creating an object matching 
+          IPv4 clients of subnet "192.0.2.0/24", as well as IPv6 clients of subnet "2001:db8::/32". 
+          Such definition results with an empty list of clients, as the constraints are additives and 
+          a client address cannot be both IPv4 and IPv6.
+      </t>
+
+        <figure>
+            <artwork><![CDATA[
+{
+  "capabilities": [
+    {
+      "capability-type": <CDNI capability object type>,
+      "capability-value": <CDNI capability object>,
+      },
+      "footprints": [
+          {
+              "footprint-type": "ipv4cidr",
+              "footprint-value": ["192.0.2.0/24"]
+          },
+          {
+              "footprint-type": "ipv6cidr",
+              "footprint-value": ["2001:db8::/32"]
+          }
+      ]
+    }
+  ]
+}
+]]>
+            </artwork>
+        </figure>
+
+
+      <t>To overcome the described limitation, and allow a list of footprint constraints that
```

Comment:
I think the "," is redundant here ("To overcome the described limitation and allow a list ...")

Reply:
As for your question "Any idea for better name for section 4.2?"  Maybe change it to:  "FCI using IPv4v6CIDR example" ?